src/finn/custom_op/fpgadataflow/__init__.py

```diff
@@ -74,6 +77,7 @@
 custom_op["Pool_Batch"] = Pool_Batch
 custom_op["FMPadding_Batch"] = FMPadding_Batch
 custom_op["Thresholding_Batch"] = Thresholding_Batch
+custom_op["Thresholding_Binary_search"] = Thresholding_Bin_Search
```

Comment:
For consistency, it would be better to name the customOp the same as the op_type. So both Thresholding_Binary_Search.

Reply:
I fully agree, and I had already changed this locally.

The only issue I am seeing with that though is the following:
"WARNING: [BD 41-1753] The name 'Thresholding_Binary_Search_0' you have specified is long. The Windows OS has path length limitations. It is recommended you use shorter names(less than 25 characters) to reduce the likelihood of issues when/if running on windows OS."

This happens during the Vivado IP stitching stage. This may be something worth ignoring as I doubt Vivado synthesis will be happening on Windows